packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -779,20 +609,6 @@ export const prepareVaultManagerKit = (
           return factoryPowers.getGovernedParams();
         },
 
-        /**
-         * In extreme situations, system health may require liquidating all vaults.
```

Comment:
is this no longer a requirement?

Reply:
The comment notwithstanding, this wasn't hooked up to an API, and I don't know how it ought to be connected to governance. If there's a future requirement for it, we can add something similar in the context of the new liquidation approach.